orchestrator/careplanservice/service.go

```diff
@@ -284,18 +279,11 @@ func (s *Service) handleGet(httpRequest *http.Request, httpResponse http.Respons
 		coolfhir.WriteOperationOutcomeFromError(err, operationName, httpResponse)
 		return
 	}
-
+	hdrs := make(map[string]string)
```

Comment:
```suggestion
	outputHeaders := make(map[string]string)
```
I know we already have `headers` but I initially wasn't sure what `hdrs` were